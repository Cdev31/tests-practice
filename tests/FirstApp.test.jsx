import { render } from "@testing-library/react";
import { FirstApp } from "../src/FirstApp";



describe('Pruebas en <FirstApp />', () => {

    test(' debe de hacer match con el snapshot', ()=>{

        const { container } = render( <FirstApp/> )

        expect( container ).toMatchSnapshot( );
    })

    test('Debe mostrar un titulo', () => {
        const title = "Hola soy vegeta"
        const { container, getByText } = render( <FirstApp title={title} /> )

        expect( getByText(title) ).toBeTruthy()

        const h2 = container.querySelector('h1')

        expect( h2.innerHTML).toContain( title)

    });

});