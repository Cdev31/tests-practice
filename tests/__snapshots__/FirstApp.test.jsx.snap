// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <FirstApp />  debe de hacer match con el snapshot 1`] = `
<div>
  <h1
    data-testid="test-title"
  >
     
    No hay título
     
  </h1>
  <p>
    cambios
  </p>
  <p>
    No hay subtítulo
  </p>
  <p>
    No hay subtítulo
  </p>
  <p>
    Fernando Herrera
  </p>
</div>
`;
